npm-packages/convex/src/bundler/index.ts

```diff
@@ -314,21 +330,30 @@ export async function bundleSchema(
 }
 
 export async function bundleAuthConfig(ctx: Context, dir: string) {
-  const authConfigPath = path.resolve(dir, "auth.config.js");
-  const authConfigTsPath = path.resolve(dir, "auth.config.ts");
-  if (ctx.fs.exists(authConfigPath) && ctx.fs.exists(authConfigTsPath)) {
+  const candidates = [
+    "auth.config.ts",
+    "auth.config.mjs",
+    "auth.config.cjs", 
+    "auth.config.js"
+  ];
+  
+  const existingPaths = candidates
+    .map(filename => path.resolve(dir, filename))
+    .filter(filepath => ctx.fs.exists(filepath));
+  
+  if (existingPaths.length > 1) {
     return await ctx.crash({
       exitCode: 1,
       errorType: "invalid filesystem data",
-      printedMessage: `Found both ${authConfigPath} and ${authConfigTsPath}, choose one.`,
+      printedMessage: `Found multiple auth config files: ${existingPaths.join(", ")}, choose one.`,
```

Comment:
this could regress some (probably broken) workflows where people compile their ts to js in-place